src/Api/PubnubApi/Pubnub.cs

```diff
@@ -42,10 +46,26 @@ public ISubscribeOperation<T> Subscribe<T>()
 		{
             if (pubnubConfig[InstanceId].EnableEventEngine)
             {
-                EndPoint.SubscribeOperation2<T> subscribeOperation = new EndPoint.SubscribeOperation2<T>(pubnubConfig.ContainsKey(InstanceId) ? pubnubConfig[InstanceId] : null, JsonPluggableLibrary, pubnubUnitTest, pubnubLog, null, tokenManager, this);
+                SubscribeEventEngine subscribeEventEngine;
+				Action<Pubnub, PNStatus> statusListener = null;
+				Action<Pubnub, PNMessageResult<T>> messageListener = null;
+				if (this.subscribeEventEngineFactory.hasEventEngine(InstanceId)) {
+                    subscribeEventEngine = subscribeEventEngineFactory.getEventEngine(InstanceId); }
+                else {
+					var subscribeManager = new SubscribeManager2(pubnubConfig[InstanceId], JsonPluggableLibrary, pubnubUnitTest, pubnubLog, telemetryManager, tokenManager, this);
+                    if (subscribeCallbackListenerList!= null && subscribeCallbackListenerList.Count > 0) {
+						messageListener = subscribeCallbackListenerList[0].Message;
+                        statusListener = subscribeCallbackListenerList[0].Status;
```

Comment:
subscribeCallbackListenerList[0].Status
Any reason for looking at only one listener?

Reply:
covered!